src/s1reader/s1_orbit.py

```diff
@@ -293,6 +300,9 @@ def get_orbit_file_from_list(zip_path: str, orbit_file_list: list) -> str:
     # extract platform id, start and end times from swath file name
     mission_id, t_swath_start_stop = get_file_name_tokens(zip_path)
 
+    # Apply temporal margin to the start time of the frame
+    t_swath_start_stop[0] = t_swath_start_stop[0] - margin_start_time
```

Comment:
The comment says "to the start time". Should it be "to the stop time"?

Reply:
Explained in the meeting, but also leaving here for the record. `t_swath_start_stop` is a list of `datetime.datetime` with two elements. The first and seconds elements in the list is starting and end time respectively.